File: library/src/main/java/me/shkschneider/skeleton/helper/SoundHelper.java
package me.shkschneider.skeleton.helper;

import android.media.Ringtone;
import android.media.RingtoneManager;
import android.net.Uri;

public class SoundHelper {

    public static Ringtone notification() {
        final Uri uri = RingtoneManager.getDefaultUri(RingtoneManager.TYPE_NOTIFICATION);
        return RingtoneManager.getRingtone(ContextHelper.applicationContext(), uri);
    }

    public static Ringtone ringtone() {
        final Uri uri = RingtoneManager.getDefaultUri(RingtoneManager.TYPE_RINGTONE);
        return RingtoneManager.getRingtone(ContextHelper.applicationContext(), uri);
    }

    public static Ringtone alarm() {
        final Uri uri = RingtoneManager.getDefaultUri(RingtoneManager.TYPE_ALARM);
        return RingtoneManager.getRingtone(ContextHelper.applicationContext(), uri);
    }

    @Deprecated // Discouraged
    public static Ringtone all() {
        final Uri uri = RingtoneManager.getDefaultUri(RingtoneManager.TYPE_ALL);
        return RingtoneManager.getRingtone(ContextHelper.applicationContext(), uri);
    }

}

File: library/src/main/java/me/shkschneider/skeleton/helper/SoundHelper.kt
package me.shkschneider.skeleton.helper

import android.media.Ringtone
import android.media.RingtoneManager

object SoundHelper {

    fun notification(): Ringtone {
        val uri = RingtoneManager.getDefaultUri(RingtoneManager.TYPE_NOTIFICATION)
        return RingtoneManager.getRingtone(ContextHelper.applicationContext(), uri)
    }

    fun ringtone(): Ringtone {
        val uri = RingtoneManager.getDefaultUri(RingtoneManager.TYPE_RINGTONE)
        return RingtoneManager.getRingtone(ContextHelper.applicationContext(), uri)
    }

    fun alarm(): Ringtone {
        val uri = RingtoneManager.getDefaultUri(RingtoneManager.TYPE_ALARM)
        return RingtoneManager.getRingtone(ContextHelper.applicationContext(), uri)
    }

    @Deprecated("All types of sounds.")
    fun all(): Ringtone {
        val uri = RingtoneManager.getDefaultUri(RingtoneManager.TYPE_ALL)
        return RingtoneManager.getRingtone(ContextHelper.applicationContext(), uri)
    }

}
